q2_gneiss/plot/_plot.py

```diff
@@ -49,56 +55,128 @@ def f(x):
         num_features = pd.DataFrame(
             {num_clade.name: taxa_df.loc[num_clade.name]}
             ).T
+        r = 1
     else:
         num_features = taxa_df.loc[num_clade.subset()]
+        r = len(list(num_clade.tips()))
 
     if denom_clade.is_tip():
         denom_features = pd.DataFrame(
             {denom_clade.name: taxa_df.loc[denom_clade.name]}
             ).T
+        s = 1
     else:
         denom_features = taxa_df.loc[denom_clade.subset()]
+        s = len(list(denom_clade.tips()))
+
+    balances = (np.log(table.loc[:, num_features.index]).mean(axis=1) -
+                np.log(table.loc[:, denom_features.index]).mean(axis=1))
+
+    balances = balances * np.sqrt(r * s / (r + s))
+    balances = pd.DataFrame(balances, index=table.index,
+                            columns=[balance_name])
```

Comment:
I don't completely understand this stuff still, but the sqrt looks like the IRL transform? Is there some code we could reuse here instead? I guess I don't really understand why the input went for `Balance` to `Composition`, but I take it something special is happening here?

Reply:
OH, it's using the taxonomy as the basis, which is the new part here, right?

Reply:
Not quite - this PR is completely independent from #18

The main motivation here is to reduce the number for free floating parameters.

We do need the `--i-table` here for the proportion plots.  Rather than having the user pass in all of the balances as an extra parameter, I was thinking that it may be easier to just compute the ilr transform that one balance that the user is interested in (since it is super fast).